core/indexer/process/interface.go

```diff
@@ -0,0 +1,22 @@
+package process
+
+import (
+	"bytes"
+
+	"github.com/elastic/go-elasticsearch/v7/esapi"
+)
+
+// DatabaseClientHandler is an interface that do requests to elasticsearch server
```

Comment:
the interface doesn't do requests. It defines the actions that a component that does request should do.

Reply:
done.